storage/migrations/17_chainvis_views_not_materialized.go

```diff
@@ -0,0 +1,145 @@
+package migrations
+
+import (
+	"github.com/go-pg/migrations/v8"
+)
+
+// Schema version 17 changes materialized views for chain_visualizer to simple views
+
+func init() {
+	up := batch(`
+DROP MATERIALIZED VIEW IF EXISTS chain_visualizer_blocks_view;
+DROP MATERIALIZED VIEW IF EXISTS chain_visualizer_blocks_with_parents_view;
+DROP MATERIALIZED VIEW IF EXISTS chain_visualizer_orphans_view;
+DROP MATERIALIZED VIEW IF EXISTS chain_visualizer_chain_data_view;
+
+CREATE VIEW chain_visualizer_chain_data_view AS
+	SELECT
+		main_block.cid AS block,
+		bp.parent AS parent,
+		main_block.miner,
+		main_block.height,
+		main_block.parent_weight AS parentweight,
+		main_block.timestamp,
+		main_block.parent_state_root AS parentstateroot,
+		parent_block.timestamp AS parenttimestamp,
+		parent_block.height AS parentheight,
```

Comment:
name as `parent_height` rather than `parentheight`?

Reply:
this is an instance of more general: "prefer '_' or camel case over... just pushing the words together"

Reply:
These views go to an externally supported app. Changing column names will require updates from the devs (something I wanted to avoid at the moment to get this change into production for them quickly).